src/tasks/withdraw/token_balances.ts

```diff
@@ -0,0 +1,36 @@
+import axios from "axios";
+
+interface AddressInfoResponse {
```

Comment:
think you should be able to do sth like this to make it more readable. Shouldn't make a difference since you are not exporting this interface (can't reuse it's sub types):
```ts
interface AddressInfoResponse {
    tokens: {
        tokenInfo: {
            address: string,
            decimals: number,
            price: {
                rate: number
            }
        },
        balance: number
    }[]
}
```